src/sql/workbench/contrib/editData/browser/media/editData.css

```diff
@@ -7,6 +7,18 @@
 	box-sizing: border-box;
 }
 
+.activePanelBorder{
+	border-top: 1px solid rgba(128, 128, 128, 0.35);
```

Comment:
As we discussed offline this should really be hooking into the color registry to get the border color instead of hardcoding it (panel.border is probably the one you want to use here)